### PR TITLE
[PyTorch] Add Transposed Convolution

### PR DESCRIPTION
**Pull request by AnirudhDagar**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1530/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1530/

File: chapter_computer-vision/transposed-conv.md [hunk 1]
```diff
@@ -82,22 +119,32 @@ tconv.initialize()
...
+
 ## Analogy to Matrix Transposition
 
 The transposed convolution takes its name from the matrix transposition. In fact, convolution operations can also be achieved by matrix multiplication. In the example below, we define a $3\times 3$ input $X$ with a $2\times 2$ kernel $K$, and then use `corr2d` to compute the convolution output.
 
 ```{.python .input}
-X = np.arange(9).reshape(3, 3)
-K = np.array([[0, 1], [2, 3]])
+#@tab all
+X = d2l.arange(9.0).reshape(3, 3)
```
Comment (astonzhang) [inline chapter_computer-vision/transposed-conv.md:136]: Is it necessary to use 9.0 in PyTorch?

Comment (astonzhang) [reply chapter_computer-vision/transposed-conv.md:136]: I am merging it first. If we can use 9, just send another pr to simplify :)

Comment (AnirudhDagar) [reply chapter_computer-vision/transposed-conv.md:136]: We can use 9, in that case, we'll need to cast the tensor to float32: `X.float()`. I can make the change if that's preferred?

Comment (astonzhang) [reply chapter_computer-vision/transposed-conv.md:136]: Then that's ok

**Pull request merged**